gpflow/kernels/periodic.py

```diff
@@ -31,26 +31,23 @@ class Periodic(Kernel):
     σ² is the variance parameter,
     γ is the period parameter.
 
-    (note that usually we have a factor of 4 instead of 0.5 in front but this
-    is absorbed into the lengthscale hyperparameter).
+    NOTE: that usually we have a factor of 4 instead of 0.5 in front but this
+        is absorbed into the lengthscale hyperparameter.
+    NOTE: periodic kernel uses `active_dims` of a base kernel, therefore
+        the constructor doesn't have it as an argument.
     """
 
-    def __init__(self, base: Stationary, period: Union[float, List[float]] = 1.0,
-                 **kwargs):
+    def __init__(self, base: Stationary, period: Union[float, List[float]] = 1.0):
         """
         :param base: the base kernel to make periodic; must inherit from Stationary
-            Note that active_dims should be specified in the base kernel.
+            Note that `active_dims` should be specified in the base kernel.
```

Comment:
This line isn't necessary

Reply:
Yes it is, users were confused (see #1230)

Reply:
ok. happy to leave as is